MAUI/Circular-Charts/DataLabels.md

```diff
@@ -222,4 +300,76 @@ this.Content = chart;
+
+## Smart Labels
+
+This feature is used to arrange the data labels smartly and avoid the intersection when there is overlapping of labels. The [SmartLabelAlignment]() enum contains the following values.
+
+* [Shift]() - smartly arranges the overlapped data labels.
+* [Hide]() - hides the intersected data labels.
+* [None]() - intersected data labels will be visible.
+
+By default, value for [SmartLabelAlignment]() property [Shift]().
+
+If the [LabelPosition]() is [Inside]() and the [SmartLabelAlignment]() is [Shift](), then the overlapped labels will shift to outside the slices and arrange smartly. If the [LabelPosition]() is [Inside]() and the [SmartLabelAlignment]() is [Hide](), then the overlapped labels will be hidden.
+
+If the [LabelPosition]() is [Outside]() and the [SmartLabelAlignment]() is [Shift](), then the overlapped labels arrange smartly. If the [LabelPosition]() is [Outside]() and the [SmartLabelAlignment]() is [Hide](), then the overlapped labels will be hidden.
+
+If the [SmartLabelAlignment]() is [None](), then the overlapped labels will be visible irrespective of LabelPosition.
+
+When the [SmartLabelAlignment]() is [Shift](), and if the data label goes out of the chart area, then the labels got trimmed and the tooltip is shown when clicking/tapping the data label. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCircularChart>
+
+    <chart:PieSeries ItemsSource="{Binding Data}" 
+                     ShowDataLabels="True"
+                     XBindingPath="Product" 
+                     YBindingPath="SalesRate">
+            <chart:PieSeries.DataLabelSettings>
+                <chart:CircularDataLabelSettings  LabelPosition="Outside" SmartLabelAlignment="Shift" >
+                    <chart:CircularDataLabelSettings.ConnectorLineSettings>
+                        <chart:ConnectorLineStyle ConnectorType="Curve"></chart:ConnectorLineStyle>
+                    </chart:CircularDataLabelSettings.ConnectorLineSettings>
+                </chart:CircularDataLabelSettings>
+            </chart:PieSeries.DataLabelSettings>             
+    </chart:PieSeries>
+
+</chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCircularChart chart = new SfCircularChart();
+. . .
+PieSeries series = new PieSeries();
+series.ItemsSource = new ViewModel().Data;
+series.XBindingPath = "Product";
+series.YBindingPath = "SalesRate";
+series.ShowDataLabels = true;
+
+var connectorLineStyle = new ConnectorLineStyle
+{
+    ConnectorType = ConnectorType = ConnectorType.Curve,
+};
+
+series.DataLabelSettings = new CircularDataLabelSettings()
+{
+    LabelPosition= ChartDataLabelPosition.Outside,
+    SmartLabelAlignment = SmartLabelAlignment.Shift,
+    ConnectorLineSettings = connectorLineStyle,
+};
+
+chart.Series.Add(series);
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![Smart Labels in MAUI Chart](DataLabel_images/maui_chart_smart_datalabel.png)
```

Comment:
@EzhilarasanElangovan31 this image is missing

Reply:
Modified the already added image